build-system/pr-check.js

```diff
@@ -102,9 +104,11 @@ function printChangeSummary() {
       colors.cyan(process.env.TRAVIS_PULL_REQUEST_SHA));
   console.log(filesChanged);
 
+  const branchPoint = process.env.TRAVIS ?
+    gitPrBranchPoint() : gitBranchPointFromMaster();
```

Comment:
This change seems unrelated. Need to update the description?

Reply:
It's part of the commit log logic. Will update.